src/coq_rules.ml

```diff
@@ -173,21 +174,36 @@ let setup_rules ~sctx ~dir ~dir_contents (s : Dune_file.Coq.t) =
             ~ml_iflags ~mlpack_rule) coq_modules in
   coq_rules
 
+(* This is here for compatibility with Coq < 8.11, which expects the
+   plugin's `.cmxs` files to be in the folder containing the `.vo`
+   files *)
+let coq_cmx_install_rules ~dst_dir libs =
+  let rules_for_lib lib =
+    Mode.Dict.get (Lib.plugins lib) Mode.Native |>
+    List.map ~f:(fun plugin_file ->
+      let dst = Path.(to_string (relative dst_dir (basename plugin_file))) in
+      None, Install.(Entry.make Section.Lib_root ~dst plugin_file))
```

Comment:
This dst argument should be a Path.Local.t eventually. It's not for this PR to fix but this string typing needs to be slowly eliminated.

Reply:
Indeed, but I guess we should defer that to its own PR.